x-pack/filebeat/input/awss3/input_integration_test.go

```diff
@@ -353,7 +358,7 @@ func TestInputRunS3(t *testing.T) {
 	assert.EqualValues(t, s3Input.metrics.s3ObjectsRequestedTotal.Get(), 7)
 	assert.EqualValues(t, s3Input.metrics.s3ObjectsListedTotal.Get(), 8)
 	assert.EqualValues(t, s3Input.metrics.s3ObjectsProcessedTotal.Get(), 7)
-	assert.EqualValues(t, s3Input.metrics.s3ObjectsAckedTotal.Get(), 6)
+	assert.EqualValues(t, s3Input.metrics.s3ObjectsAckedTotal.Get(), 7)
```

Comment:
Why did the number of acked objects changed?

Reply:
because this should be 7 to begin with, right??? 

1. "testdata/events-array.json",
2. "testdata/invalid.json",
3. "testdata/log.json",
4. "testdata/log.ndjson",
5. "testdata/multiline.json",
6. "testdata/multiline.json.gz",
7. "testdata/multiline.txt"